src/tests/multihost/alltests/test_misc_proxy.py

```diff
@@ -0,0 +1,392 @@
+        execute_cmd(multihost, "getent netgroup QAeng")
+        execute_cmd(multihost, 'echo "" > /etc/netgroup')
+
+    def test_bz647816(self, multihost, backupsssdconf):
+        """
+        :title: More than 10 auth attempt times out bz647816
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=647816
+        :id: 201a44c0-b252-11ec-94b7-845cf3eff344
+        :steps:
+            1. Auth a user more than 12 times
+            2. Search for Error
+        :expectedresults:
+            1. Should succeed
+            2. Should not succeed
+        """
+        # more than 10 auth attempt times out bz647816
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'proxy_lib_name': 'ldap',
+                         'debug_level': '0xFFF0'}
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        client_e = multihost.client[0].ip
+        for _ in range(12):
+            ssh1 = SSHClient(client_e, username="foo1@example1",
+                             password="Secret123")
+            ssh1.close()
+        with pytest.raises(subprocess.CalledProcessError):
+            execute_cmd(multihost, "grep 'All available child slots are full, "
+                                   "queuing request' /var/log/sssd/*")
+
+    def test_bz871424(self, multihost, backupsssdconf):
+        """
+        :title: authconfig chokes on sssd.conf with chpass_provider directive
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=871424
+        :id: ce5fe4d2-e6e0-11ec-af7d-845cf3eff344
+        :steps:
+            1. Configure SSSD chpass_provider = proxy
+            2. Run authconfig --test
+        :expectedresults:
+            1. Should succeed
+            2. Should succeed
+        """
+        # more than 10 auth attempt times out bz647816
+        tools = sssdTools(multihost.client[0])
+        domain_name = tools.get_domain_section_name()
+        domain_params = {'id_provider': 'ldap',
+                         'auth_provider': 'ldap',
+                         'chpass_provider': 'proxy',
+                         'proxy_pam_target': 'sssdproxyldap',
+                         'ldap_schema': 'rfc2307',
+                         'enumerate': 'false',
+                         'cache_credentials': 'true' }
+        tools.sssd_conf('domain/' + domain_name, domain_params)
+        tools.clear_sssd_cache()
+        execute_cmd(multihost, "authselect test sssd")
+
+    def test_bz1221992(self, multihost, backupsssdconf):
+        """
+        :title: sssd_be segfault at 0 ip sp error 6 in libtevent.so.0.9.21
+        :bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1221992
+        :id: a97bf86a-e6e3-11ec-a1ee-845cf3eff344
+        :steps:
+            1. Configure user=sssd  is in in sssd.conf
+            2. user should be able to change the
+             password without segfault
```

Comment:
Indent this line to start from under user

Reply:
Fixed